advocacy_docs/partner_docs/QuestFoglightforDatabases/03-SolutionSummary.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: 'Solution Summary'
+description: 'Brief explanation of the solution and its purpose'
+---
+Quest Foglight for databases is a tool that enables monitoring of health and performance issues for all database platforms, whether on-premise, hybrid, or in the cloud.  This tool enables the user to collect their database information in one place and monitor them simultaneously. It offers 24/7 unattended data collection with a straightforward deployment for monitoring.
```

Comment:
"databases" should be upper-case

Reply:
This first sentence should be re-phrased, it is too similar to the product description in Partner Information

Reply:
> This first sentence should be re-phrased, it is too similar to the product description in Partner Information

"Whether your database sits on-premise, in the cloud, or hybrid, Quest Foglight for Databases offers continuous monitoring for health and performance metrics."
Would something phrased like this be less redundant?

Reply:
Yes that would be good.